src/photos/ducks/clustering/matching.js

```diff
@@ -58,3 +60,27 @@ export const matchingClusters = (photo, albums) => {
   }
   return []
 }
+
+/**
+ * Find the relevant parameters for the given set of photos.
+ * @param {Object[]} params - the array of parameters including their period
+ * @param {Object[]} photos - the array of photos including their timestamps
+ */
+export const matchingParameters = (parameters, photos) => {
+  // Take the average date in the photos to compare with parameters periods.
+  const datetime = averageDate(photos)
+
+  const lastParams = parameters[parameters.length - 1]
+  if (new Date(lastParams.period.end) <= datetime) {
+    // The date is newer than the last parameters
+    return lastParams
+  } else if (new Date(parameters[0].period.start) >= datetime) {
+    // The date is older than the first parameters
+    return parameters[0]
+  } else {
+    // The date is inside the parameters periods
+    return parameters.find(param => {
+      return new Date(param.period.end) >= new Date(datetime)
```

Comment:
`datetime ` is already a `Date` object ;)

```suggestion
      return new Date(param.period.end) >= datetime
```